<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CryptoView</class>
 <widget class="QMainWindow" name="CryptoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CryptoView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(206,159,252), stop:1 rgba(115,103,240));
background: qlineargradient(spread:pad, x1:0, y1:0.25, x2:1, y2:0.858, stop:0 rgba(206,159,252), stop:1 rgba(115,103,240))</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>25</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>25</number>
     </property>
     <property name="spacing">
      <number>-1</number>
     </property>
     <item row="1" column="0">
      <widget class="QGroupBox" name="crytoBox_3">
       <property name="styleSheet">
        <string notr="true">.QGroupBox {
	background:rgba(0,0,0,0.1);
    border-width: 1px;
    border-style: solid;
	border-color: rgba(255,255,255,0.5);
	border-radius: 5px
 }</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="coin3_title">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Helvetica Neue'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,sans-serif'; font-size:24px; font-weight:496; color:#ffffff;&quot;&gt;LOADING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="coin3_img">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>221</width>
          <height>131</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/resources/white-logos/gbyte  .png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="coin3_value">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rbga(0,0,0,0)</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;--&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="coin3_btn">
        <property name="geometry">
         <rect>
          <x>-30</x>
          <y>-10</y>
          <width>271</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="crytoBox_4">
       <property name="styleSheet">
        <string notr="true">.QGroupBox {
	background:rgba(0,0,0,0.1);
    border-width: 1px;
    border-style: solid;
	border-color: rgba(255,255,255,0.5);
	border-radius: 5px
 }</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="coin4_title">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Helvetica Neue'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,sans-serif'; font-size:24px; font-weight:496; color:#ffffff;&quot;&gt;LOADING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="coin4_img">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>221</width>
          <height>131</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/resources/white-logos/gbyte  .png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="coin4_value">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rbga(0,0,0,0)</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;--&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="coin4_btn">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>-10</y>
          <width>291</width>
          <height>491</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="crytoBox_1">
       <property name="styleSheet">
        <string notr="true">#crytoBox_1 {
	background:rgba(0,0,0,0.1);
    border-width: 1px;
    border-style: solid;
	border-color: rgba(255,255,255,0.5);
	border-radius: 5px
 }
#cryptoBox_1:hover {
	background:rgba(0,0,0,0.8);
 }
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="coin1_title">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,sans-serif'; font-size:24px; font-weight:496; color:#ffffff;&quot;&gt;LOADING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="coin1_img">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>221</width>
          <height>131</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/resources/white-logos/gbyte  .png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="coin1_value">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rbga(0,0,0,0)</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;--&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="coin1_btn">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>-10</y>
          <width>241</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="crytoBox_2">
       <property name="styleSheet">
        <string notr="true">.QGroupBox {
	background:rgba(0,0,0,0.1);
    border-width: 1px;
    border-style: solid;
	border-color: rgba(255,255,255,0.5);
	border-radius: 5px
 }</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="coin2_title">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Helvetica Neue'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,sans-serif'; font-size:24px; font-weight:496; color:#ffffff;&quot;&gt;LOADING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="coin2_img">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>221</width>
          <height>131</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/resources/white-logos/gbyte  .png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="coin2_value">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rbga(0,0,0,0)</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;--&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="coin2_btn">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>-10</y>
          <width>241</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
